### PR TITLE
chore: bump version for November release

### PR DESCRIPTION
**Pull request by prusnak**
This is something we forgot to do after forking the `release/2020-10` branch from the develop.

We'll always keep the version `YY.MM.0` in develop where `MM` is the upcoming month. More info in the updated `docs/misc/versioning.md` file

### CODE REVIEW

**Review by tsusanka — APPROVED** [review]
LGTM, but I can't tell if this is all the places, someone from the Suite team should have a look.

I will cherry-pick this to release/2020-10-beta then to update the versions there.

**Review by slowbackspace — APPROVED** [review]

**Pull request merged**
